  # Weights for the deviation from the reference position
  x_distance_weight: 100.0
  y_distance_weight: 100.0
  z_distance_weight: 100.0
  # Weight for the deviation from the reference orientation
  orientation_weight: 100.0
  # Cost for leafing the field
  leafing_field_cost: 10000.0
  # Negative and positive limits of the field
  field_limit_x_min: -30.0
  field_limit_x_max: 30.0
  field_limit_y_min: -30.0
  field_limit_y_max: 30.0
  field_limit_z_max: 30.0
  # Floor Cost
  floor_cost: 10000
  # Cost for Obstacle colision
  obstacle_cost: 10000
  # Position of the Obstacle
  obstacle_position_x: 5
  obstacle_position_y: 5
